examples/webgl_postprocessing_ssao.html

```diff
@@ -0,0 +1,222 @@
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				renderer = new THREE.WebGLRenderer( { antialias: false } );
+				renderer.setClearColor( 0xa0a0a0 );
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( renderer.domElement );
+				var depthShader = THREE.ShaderLib[ "depthRGBA" ];
+				var depthUniforms = THREE.UniformsUtils.clone( depthShader.uniforms );
+
+				depthMaterial = new THREE.ShaderMaterial( { fragmentShader: depthShader.fragmentShader, vertexShader: depthShader.vertexShader,
+					uniforms: depthUniforms, blending: THREE.NoBlending } );
+								
+				camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 100, 700 );
+				camera.position.z = 500;
+
+				scene = new THREE.Scene();
+
+				group = new THREE.Object3D();
+				scene.add( group );
+		   
+				var geometry = new THREE.IcosahedronGeometry( 5, 1 );
+				for ( var i = 0; i < 200; i ++ ) {
+				
+					var material = new THREE.MeshBasicMaterial();
+					material.color.r = Math.random();
+					material.color.g = Math.random();
+					material.color.b = Math.random();
+				   
+					var mesh = new THREE.Mesh( geometry, material );
+					mesh.position.x = Math.random() * 400 - 200;
+					mesh.position.y = Math.random() * 400 - 200;
+					mesh.position.z = Math.random() * 400 - 200;
+					mesh.rotation.x = Math.random();
+					mesh.rotation.y = Math.random();
+					mesh.rotation.z = Math.random();
+
+					mesh.scale.x = mesh.scale.y = mesh.scale.z = Math.random() * 10 + 1;
+					group.add( mesh );
+				}
+				
+				stats = new Stats();
+				stats.domElement.style.position = 'absolute';
+				stats.domElement.style.top = '0px';
+				container.appendChild( stats.domElement );
+
+				// postprocessing                
+				initPostprocessing();
+			
+				// Init gui
+				var gui = new dat.GUI();
+				gui.add( postprocessing, "enabled" ).onChange();
+				gui.add( postprocessing, "renderMode", { framebuffer: 0, onlyAO: 1 } ).onChange( renderModeChange ).listen();
+
+				window.addEventListener( 'resize', onWindowResize, false );
+			}
+
+			function renderModeChange( value ) {
+
+				if ( value == 0 ) { // framebuffer
+					postprocessing.ssao_uniforms[ 'onlyAO' ].value = false;
+				} else if ( value == 1 ) {  // onlyAO
+					postprocessing.ssao_uniforms[ 'onlyAO' ].value = true;
+				} else {
+					console.error( "Not define renderModeChange type: " + value );
+				}
+			}
+			
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+				initPostprocessing();
+			}
+
+			function initPostprocessing() {
+				
+				postprocessing.scene = new THREE.Scene();
+				postprocessing.camera = new THREE.OrthographicCamera( window.innerWidth / - 2, window.innerWidth / 2,  window.innerHeight / 2, window.innerHeight / - 2, -1000, 1000 );
```

Comment:
This can be done without making `orthographicCamera` dependent on the window size. `EffectComposer` does so. I think it is good that you have worked to understand the details, but `EffectComposer` has already figured out how to address these issues.